package.json

```diff
@@ -1,41 +1,76 @@
 {
-  "name": "ganache-cli",
-  "version": "6.9.1",
-  "bin": {
-    "ganache-cli": "cli.js"
-  },
-  "main": "./build/ganache-core.node.cli.js",
-  "scripts": {
-    "start": "node cli.js",
-    "build": "webpack-cli --config ./webpack/webpack.node.config.js",
-    "docker": "docker build -t trufflesuite/ganache-cli . && docker run -p 8545:8545 trufflesuite/ganache-cli",
-    "prepublishOnly": "npm run build"
-  },
-  "dependencies": {
-    "ethereumjs-util": "6.1.0",
-    "source-map-support": "0.5.12",
-    "yargs": "13.2.4"
-  },
-  "bundleDependencies": [
-    "source-map-support",
-    "yargs",
-    "ethereumjs-util"
-  ],
-  "devDependencies": {
-    "ganache-core": "2.10.2",
-    "js-scrypt": "0.2.0",
-    "node-loader": "^0.6.0",
-    "on-build-webpack": "^0.1.0",
-    "prepend-file": "^1.3.1",
-    "scrypt": "6.0.3",
-    "sha3": "1.2.2",
-    "shebang-loader": "0.0.1",
-    "webpack": "^4.18.0",
-    "webpack-cli": "3.1.0"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/trufflesuite/ganache-cli"
-  },
-  "license": "MIT"
+	"name": "ganache-cli",
```

Comment:
Let's keep indentation at 2 spaces here (though 4 is common for json)

Reply:
Not sure what's happening - tab indents are at 2 on my machine.